.github/workflows/build.yml

```diff
@@ -66,6 +90,6 @@ jobs:
     - name: Upload artifacts
       uses: actions/upload-artifact@v3
       with:
-        name: pc_ble_driver_py-linux-${{ matrix.os }}-${{ matrix.toxpy}}
+        name: pc_ble_driver_py-${{ matrix.os }}-${{ matrix.toxpy}}
```

Comment:
This might break things / require more effort:

![image](https://github.com/embedded-community/pc-ble-driver-py/assets/26225854/61b584b4-33a6-4581-9d7b-e7697497795f)
https://pypi.org/project/pc-ble-driver-py/#files

Reply:
Yeah we can check this at the next step.